### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by mogulano**
Please provide an email where to report security sensitive issues.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities to xxxxxxxxx@xxxxxxxxxxxxxx.xxx
```
Comment (jhamman) [inline SECURITY.md]: pangeo-steering-council@googlegroups.com ???

Not sure what to do here actually.

**Comment by rabernat** [comment]
Is this a legit PR or a bot?

**Comment by mogulano** [comment]
I'm not a bot. Do you confirm pangeo-steering-council@googlegroups.com as a contact before I send the security report to potentially big audience?

I would prefer the PR to be updated with email and merged by the owners of the repository before doing it. If you prefer you may obfuscate @ to prevent bots from scraping it.

File: SECURITY.md (hunk 1 above)
Comment (rabernat) [reply SECURITY.md]:
```suggestion
Please report security vulnerabilities to pangeo-steering-council@googlegroups.com 
```

**Comment by rabernat** [comment]
Thanks @mogulano for this.

**Pull request merged**